addons/crm/models/crm_lead.py

```diff
@@ -2640,3 +2676,81 @@ def _pls_get_lead_pls_values(self, domain=None):
                         lead_values.append(('tag_id', tag.id))
                 leads_values_dict[lead.id] = {'values': lead_values, 'team_id': lead['team_id'].id}
             return leads_values_dict
+
+    # PLS Backend Tooltip
+    # -------------------
+    def prepare_pls_tooltip_data(self):
+        '''
+            Compute and return all necessary information to render CrmPlsTooltip, displayed when
+            pressing the small AI button, located next to the label of probability when automated,
+            in the crm.lead form view. This method first replaces ids with display names of relational
+            fields before returning data, then also recomputes probabilities and writes them on self.
+
+            :returns dict: {
+                low_3_data: list of field-value couples for lowest 3 criterions, lowest first
+                probability: numerical value, used for display on tooltip
+                team_name: string, name of lead team if any
+                top_3_data: list of field-value couples for top 3 criterions, highest first
+            }
+        '''
+        if not self:
+            return False
```

Comment:
would perhaps replace this by `self.ensure_one()`

Reply:
Ok! I remove the `if ! tooltipData return false` in the js then :)